benchmark/syclblas/blas1/rotmg.cpp

```diff
@@ -0,0 +1,149 @@
+/***************************************************************************
+ *
+ *  @license
+ *  Copyright (C) 2022 Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCL-BLAS: BLAS implementation using SYCL
+ *
+ *  @filename rotmg.cpp
+ *
+ **************************************************************************/
+
+#include "../utils.hpp"
+
+template <typename scalar_t>
+std::string get_name() {
+  std::ostringstream str{};
+  str << "BM_Rotmg<" << blas_benchmark::utils::get_type_name<scalar_t>() << ">/";
+  return str.str();
+}
+
+template <typename scalar_t>
+void run(benchmark::State& state, ExecutorType* executorPtr, bool* success) {
+  // Create data
+  constexpr size_t param_size = 5;
+  std::vector<scalar_t> param = std::vector<scalar_t>(param_size);
+  scalar_t d1 = blas_benchmark::utils::random_data<scalar_t>(1)[0];
+  scalar_t d2 = blas_benchmark::utils::random_data<scalar_t>(1)[0];
+  scalar_t x1 = blas_benchmark::utils::random_data<scalar_t>(1)[0];
+  scalar_t y1 = blas_benchmark::utils::random_data<scalar_t>(1)[0];
+
+  ExecutorType& ex = *executorPtr;
+
+  auto buf_d1 = utils::make_quantized_buffer<scalar_t>(ex, d1);
+  auto buf_d2 = utils::make_quantized_buffer<scalar_t>(ex, d2);
+  auto buf_x1 = utils::make_quantized_buffer<scalar_t>(ex, x1);
+  auto buf_y1 = utils::make_quantized_buffer<scalar_t>(ex, y1);
+  auto buf_param = utils::make_quantized_buffer<scalar_t>(ex, param);
+
+#ifdef BLAS_VERIFY_BENCHMARK
+
+
+
```

Comment:
There are a few places where I feel like the number of blank lines is excessive.

Reply:
Agreed. Will try to remove a few of this situations

Reply:
I think I addressed this. Let me know if I missed something